CMSIS/Core/Include/core_cm23.h

```diff
@@ -92,7 +92,7 @@
 
 #elif defined ( __ICCARM__ )
   #if defined __ARMVFP__
-    #error "Compiler generates FPU instructions for a device without an FPU (check __FPU_PRESENT)"
+    #warning "Compiler generates FPU instructions for a device without an FPU (check __FPU_PRESENT)"
```

Comment:
This is now unaligned with the other compilers. Can you please give a short justification why usage of FPU instructions on a device without FPU should be a warning instead of an error?
I expect images created with wrong FPU settings to instantaneously crash at run-time, i.e. as soon as the first FPU instruction is hit.